pyo3-ffi/src/datetime.rs

```diff
@@ -487,7 +487,8 @@ extern "C" {
     pub fn PyDateTime_DATE_GET_MICROSECOND(o: *mut PyObject) -> c_int;
     #[link_name = "PyPyDateTime_GET_FOLD"]
     pub fn PyDateTime_DATE_GET_FOLD(o: *mut PyObject) -> c_int;
-    // skipped PyDateTime_DATE_GET_TZINFO (not in PyPy)
+    #[link_name = "PyPyDateTime_DATE_GET_TZINFO"]
```

Comment:
Similarly we should mark this and below as `#[cfg(Py_3_10)]` given the symbols aren't available on earlier versions.